### ISSUE TITLE
Attempting to generate from an MRS object needs a better error message

### ISSUE THREAD

**Issue by goodmami** [delph-in/pydelphin#350]
`ACEGenerator` expects a string MRS to generate from. When working with PyDelphin's MRS objects, it may be tempting to try and generate from the MRS object without first serializing to a string. The error message for this, however, is not intuitive:

```pycon
>>> from delphin import ace
>>> parser = ace.ACEParser('../erg-2018.dat')
>>> generator = ace.ACEGenerator('../erg-2018.dat', cmdargs=['-r', 'root_frag'])
>>> response = parser.interact('the phone')
>>> gresponse = generator.interact(response.result(0).mrs())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/goodmami/delphin/pydelphin/delphin/ace.py", line 255, in interact
    validated = self._validate_input(datum)
  File "/home/goodmami/delphin/pydelphin/delphin/ace.py", line 387, in _validate_input
    return _possible_mrs(datum)
  File "/home/goodmami/delphin/pydelphin/delphin/ace.py", line 624, in _possible_mrs
    for i, c in enumerate(s):
  File "/home/goodmami/delphin/pydelphin/delphin/sembase.py", line 164, in __getitem__
    return self._pidx[id]
KeyError: 0
```

The `interact()` method should perhaps through an error if the argument is not a string.

See this [discourse thread](https://delphinqa.ling.washington.edu/t/problems-with-using-pydelphin-to-generate-from-an-mrs/814).

**Comment by KelsonXie** [comment]
Hey, I would like to attempt to fix this issue, any recommendations on what the error message should be?

**Comment by goodmami** [comment]
This was closed by #351
